src/dr_llm/llm/catalog/fetchers/google.py

```
@@ -1,28 +1,28 @@
 
 def fetch_google_models(
-    adapter: GoogleAdapter,
+    provider: GoogleProvider,
 ) -> tuple[list[ModelCatalogEntry], dict[str, Any]]:
-    key = adapter.config.api_key or api_key_from_env(adapter.config.api_key_env)
+    key = provider.config.api_key or api_key_from_env(provider.config.api_key_env)
     if not key:
         raise ProviderSemanticError(
-            f"Missing Google API key for catalog sync. Set {adapter.config.api_key_env}"
+            f"Missing Google API key for catalog sync. Set {provider.config.api_key_env}"
         )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)